src/lib/core/gestures/MdGestureConfig.ts

```diff
@@ -4,16 +4,25 @@ import {HammerGestureConfig} from '@angular/platform-browser';
 /* Adjusts configuration of our gesture library, Hammer. */
 @Injectable()
 export class MdGestureConfig extends HammerGestureConfig {
+  private _hammer = typeof window !== 'undefined' ? (window as any).Hammer : null;
```

Comment:
Rather than making it `any`, could you add a `GestureManager` interface that just copies the members of `HammerManager` that we're using?